src/components/AnalyticsFetcher.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import './analytics-fetcher.css';
+import {connect} from "react-redux";
+
+class AnalyticsFetcher extends Component {
+    render() {
+        return (
+            <div>
+                <label>
+                    Video Analytics:
+                </label>
+                <pre>
+                {JSON.stringify(this.props.analyticData, null, 2)}
+                </pre>
+            </div>
+        );
+    }
+}
+
+const mapStateToProps = (state) => {
+    return {
+        analyticData:state.analyticData,
+        selectedVideo: state.selectedVideo,
```

Comment:
does this component need selectedVideo and videoIds yet?

Reply:
I guess it only needs analyticData.